integreat_cms/cms/templates/content_language_switcher.html

```diff
@@ -0,0 +1,47 @@
+<!--generic template for language switcher-->
```

Comment:
```suggestion
<!--generic template for language switcher in content forms -->
```
I'd also rename this file to `content_form_language_switcher.html` to point out that's intended to be used in forms.